tests/v0_tests/test_model_cache_management.py

```diff
@@ -0,0 +1,53 @@
+import copy
+import math
+import pprint
+import random
+import requests
+from marqo.client import Client
+from marqo.errors import MarqoApiError, MarqoError, MarqoWebError
+from tests.marqo_test import MarqoTestCase
+from marqo import enums
+from unittest import mock
+
+
+class TestModelCacheManagement(MarqoTestCase):
+
+    # NOTE: The cuda should already have model loaded in the startup
+    def setUp(self) -> None:
+        self.client = Client(**self.client_settings)
+        self.index_name = "test_index"
+        self.MODEL = "ViT-L/14"
+
+    def test_get_cuda_info(self) -> None:
+        try:
+            self.client.get_cuda_info()
+        except MarqoWebError: # catch error if no cuda device in marqo
+            pass
+
+    def test_get_cpu_info(self) -> None:
+        self.client.get_cpu_info()
+
+
+    def test_get_loaded_models(self) -> None:
+        self.client.get_loaded_models()
+
+
+    def test_eject_no_cached_model(self) -> None:
+        try:
+            self.client.eject_model("void_model", "void_device")
```

Comment:
What does this test do?

Reply:
try to eject a model that is not cached. comments added.